gradle.properties

```diff
@@ -12,6 +12,14 @@ org.gradle.jvmargs=-Xmx1536m
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
+# Disable Gradle Android plugin features.
+# Reference: https://developer.android.com/studio/releases/gradle-plugin#4-0-0-new
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
This is the most important line. There is no need to ship the generated `BuildConfig` to consumers since they have their own ones which reflect application version, package and user fields better.

The remaining is a bonus of disabling irrelevant tasks which take non-zero time to execute.